specify/specify7
# Correct form padding and round corners in Firefox

Fixes #3189

**Instructions to test:**
- Make sure that all areas with scrollbars display properly in Firefox and Chrome.
- Make sure that corners on elements with scrollbars are still rounded
- Make sure that padding is equal on both the left and right on forms

**It shouldn't look like this:**

<img width="200" alt="image" src="https://user-images.githubusercontent.com/37256050/225700789-a3b1789c-fc11-4b34-af11-91d791a6ad84.png">

<img width="500" alt="image" src="https://user-images.githubusercontent.com/37256050/225700847-043dd0b1-dd22-424b-b8a7-594590215a80.png">
